coralproject/talk
# feat/CORL-2244 StoryInfoDrawer

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2244 by creating a "Story Info Drawer" where admins can view information about a story as well as take quick actions such as updating it's status, trigger a rescrape, etc...

NOTES: 
- Need guidance on what UI interaction should open the drawer (currently triggered by clicking anywhere in the row)
- Could use design guidance on "Additional settings" form at bottom
- Relay seems to be caching responses on future page loads, so changes don't appear persistent to the user
- Not sure if on submissing, we should "trigger a refresh" or just update the form with the selected values and mark the form as pristine again

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
1. Navigate to admin > stories
2. Click on a story (currently anywhere in the row) and see that the drawer opens
3. Toggle a stories status, and refresh to see that it has been updated
4. Trigger a rescrape of the story
5. Update a story's mode (LIVE, QA, etc), moderation mode, and other settings
6. Put a story into QA mode and observe that you can add/remove experts from the story

## How do we deploy this PR?
No special considerations should need to be taken
